cf/regrid/regrid.py

```diff
@@ -10,19 +10,33 @@
 from ..units import Units
 from .regridoperator import RegridOperator
 
+# ESMF renamed its Python module to `esmpy` at ESMF version 8.4.0. Allow
+# either for now for backwards compatibility.
+esmpy_imported = False
 try:
-    import ESMF
-except Exception:
-    ESMF_imported = False
-else:
-    ESMF_imported = True
+    # Take the new name to use.
+    import ESMF as esmpy
+
+    esmpy_imported = True
+except ImportError:
+    pass  # avoid nested try/except...
```

Comment:
```suggestion
    pass  # avoid nested try/except...
```
Is avoiding these considered good practice? Would nesting mean we don't need the flake8 exception?

Reply:
Well, I would say that potentially importing esmpy twice is curious (even though the second import would be very fast), so I would vote for the nesting. You're call, though!

Reply:
Sounds good to me. I'll update it to the nested one, then, in a new commit which I'll push up shortly.